content/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,234 @@
+kind: Deployment
+metadata:
+  labels:
+    app: nginx-app
+  name: cfe-nginx-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-app
+  template:
+    metadata:
+      labels:
+        app: nginx-app
+    spec:
+      volumes:
+      - name: webdata
+        emptyDir: {}
+      initContainers:
+      - name: web-content
+        image: busybox
+        volumeMounts:
+        - name: webdata
+          mountPath: "/webdata"
+        command: ["/bin/sh", "-c", 'echo "<h1> Hi, this is the sample <font color=blue>Nginx App</font> deployed as the Load Balancer service type !</h1>" > /webdata/index.html']
+      containers:
+      - image: nginx
+        name: nginx
+        volumeMounts:
+        - name: webdata
+          mountPath: "/usr/share/nginx/html"
+
+$ kubectl apply -f nginx-deployment.yaml 
+service/cfe-nginx-app created
+deployment.apps/cfe-nginx-app created
+```
+
+You can check the Nginx a﻿pplication deployment b﻿y typing the following command, using the label *app=nginx-app*, and confirm all pods and services are in running states. For the sample service *cfe-nginx-app*, you should see it’s been deployed as the *LoadBalancer* type and an IP address, *172.16.17.250*, gets assigned as its *EXTERNAL-IP* :
+
+```markdown
+$ kubectl get all -l app=nginx-app
+NAME                                 READY   STATUS    RESTARTS   AGE
+pod/cfe-nginx-app-66cb4f5bbf-4nfw5   1/1     Running   0          3m20s
+
+NAME                    TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)        AGE
+service/cfe-nginx-app   LoadBalancer   10.98.244.64   172.16.17.250   80:31631/TCP   3m22s
+
+NAME                            READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/cfe-nginx-app   1/1     1            1           3m21s
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/cfe-nginx-app-66cb4f5bbf   1         1         1       3m22s
+```
+
+T﻿o verify the deployed Nginx application is working, l﻿aunch your web browser, type the string "http://" followed by the IP address *'172.16.17.250'* that's assigned t﻿o the Nginx service, then press the *Enter* key. You should see the following texts showing in the browser:
+
+*﻿Hi, this is the sample <font color=blue>Nginx App</font> deployed as the Load Balancer service type !*
+ 
+![](/img/web-nginx-app.png)
+
+
+
+
+### Summary
+
+This blog post describes the detailed process used to deploy and set up MetalLB to support customers to configure load balancers for the K8s clusters in HPE GreenLake for Private Cloud Enterprise. By deploying load balancing configuration, it allows customers to easily work together with Kubernetes Ingress for different traffic routing support of their deployed workloads in the K8s clusters. It also supports deploying applications by passing the customer certificates through their own authority. It unblocks a list of potential use cases and enhances HPE GreenLake product with additional flexibilities.
+

```

Comment:
I find this sentence "By deploying load balancing configuration, it allows customers to easily work together with Kubernetes Ingress for different traffic routing support of their deployed workloads in the K8s clusters" a bit long and unclear. Maybe you could rephrase.